modules/nf-core/bcftools/concat/main.nf

```diff
@@ -8,23 +8,32 @@ process BCFTOOLS_CONCAT {
         'biocontainers/bcftools:1.17--haef29d1_0' }"
 
     input:
-    tuple val(meta), path(vcfs), path(tbi)
+    tuple val(meta), path(vcfs), path(tbis)
+    path(bed)
 
     output:
-    tuple val(meta), path("*.gz"), emit: vcf
-    path  "versions.yml"         , emit: versions
+    tuple val(meta), path("*.${extension}") , emit: vcf
```

Comment:
Good point, but I would maybe do this change for all other modules in another PR (since all other modules where this extension calculation happen also have `vcf` as output name)